test-cargo-miri/subcrate/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+#[cfg(test)]
+compile_error!("Miri should not touch me");
```

Comment:
Did you confirm that regular `cargo test` shows this error?

Reply:
Yes, it does. Should I add a test?

Reply:
No it's fine, thanks!